### ISSUE TITLE
Suggestion: Add allergens for specific individuals

### ISSUE THREAD

**Issue by ross-holloway94** [openfoodfacts/openfoodfacts-androidapp#633]
I would like to know people's opinions on this.
I believe that adding allergens for individuals, and therefore showing who is safe to eat a product and who is not, would improve the user experience.

**Comment by teolemon** [comment]
Yes. You mean turning the current allergen detector into a multiuser experience ?
```Hélène: safe
Marianne: safe
Pierre: contains peanut ```

**Comment by ross-holloway94** [comment]
Exactly this. I wondered if it was too complicated to add this, however.

**Comment by teolemon** [comment]
I'm wondering if the case with different allergens for the various children occurs that often. 
So in terms of roadmap, not sure, in terms of storytelling ("selling the app"): it tells a great story, even to people who will never use the feature.

**Comment by ross-holloway94** [comment]
Well, in my case - my mum is allergic to nuts and white fish. I am allergic to both of those, but I am allergic to egg as well.

**Comment by teolemon** [comment]
In that case, go ahead. One motivated dev makes a strong case ;-)

**Comment by ross-holloway94** [comment]
This issue would involve:

- Storing allergens for individuals locally.
- Creating the UI to represent this data.
- Showing who should not consume the product in the Product pages.

**Comment by ross-holloway94** [comment]
I have created a Trello board for this now - https://trello.com/b/eP8zz1bg/open-food-facts-633

**Comment by teolemon** [comment]
@ross-holloway94 Was wondering the status of this. Is there anything we can salvage from the branch for the future ?

**Comment by ross-holloway94** [comment]
@teolemon sorry for the late reply. I don't think I'll have time to work on this any more, sorry. The library is available on my profile still if anybody would to like to take it on.
